Emby.Server.Implementations/HttpServer/HttpResultFactory.cs

```diff
@@ -255,16 +255,20 @@ private static string GetCompressionType(IRequest request)
         {
             var acceptEncoding = request.Headers[HeaderNames.AcceptEncoding].ToString();
 
-            if (string.IsNullOrEmpty(acceptEncoding))
+            if (!string.IsNullOrEmpty(acceptEncoding))
             {
-                //if (_brotliCompressor != null && acceptEncoding.IndexOf("br", StringComparison.OrdinalIgnoreCase) != -1)
+                // if (_brotliCompressor != null && acceptEncoding.IndexOf("br", StringComparison.OrdinalIgnoreCase) != -1)
                 //    return "br";
```

Comment:
any reason for us to _not_ support `brotli`?

Reply:
No support? I’m not 100% sure

Reply:
I think this can just be ignored until the new API is enabled. There we can enable whatever compression we want